### PR TITLE
Add a feather alternative to csv when loading parquet format files

### PR DESCRIPTION
**Pull request by tnederlof**
I got a working example (at least on my Mac setup) using feather files instead of CSV files. The advantage to that is I dont get any warnings (whereas the CSV method generates all sorts of nasty warnings although the file does come out). It would be good to test both of these methods further though.

A quick test showed feather loaded the response data project into R about 5 sec while CSV loaded in about 7.5 sec.

Also I started to add package_path variables in case people dont install packages to ~/quilt_packages. Just an idea, not sure what you @stillmatic think about that?

This was my first stab but one thing I am pretty used to is writing documentation/function flows so later on I could take a stab at standardizing, etc

### CODE REVIEW

**Comment by akarve** [comment]
Cool. Re: `quilt_packages`, `quilt install` will always generate that directory in the current working directory. `import` and `ls` walk the directory tree upwards looking for matches. We can either punt to the Python client or pick up this directory resolution policy in https://github.com/stillmatic/quiltr/issues/2.

**Comment by tnederlof** [comment]
Interesting, I just added the variable so that if different people are testing and want to move around based on their setting they can. The default I left at ~/quilt_packages/

Feather seems to work pretty well as far as I can tell (out of Python then into R). Will need to do it with data I am familiar with type wise to really ensure it all gets captured correctly.

File: R/core.R [hunk 1]
```diff
@@ -40,8 +41,8 @@ qsearch <- function(str) {
 #' @examples
 #' qpeek("akarve/examples")
 #' qpeek("akarve/examples", TRUE)
-qpeek <- function(str, robust=FALSE) {
-    path <- paste0("~/quilt_packages/", str, ".json")
+qpeek <- function(str, robust=FALSE, package_path = "~/quilt_packages/") {
```
Comment (stillmatic) [inline R/core.R:44]: If we're going to have a default search path, it should be set as an R option, so if a user has a non-standard directory, they don't need to specify it every time they use the command. This also will help reproducibility and code sharing.

See https://github.com/hadley/devtools/blob/master/R/zzz.r for an example of how to set up the options.

File: R/io.R [hunk 2]
```diff
@@ -26,15 +30,15 @@ check_package <- function(pkg, file = NULL) {
 #' qload("examples/wine", "quality")
 #' qload("examples/wine", "quality/red")
 #' qload("akarve/seattle_911", "responses")
-qload <- function(pkg, file, ...) {
+qload <- function(pkg, file, file_type = "feather", package_path = "~/quilt_packages/") {
```
Comment (stillmatic) [inline R/io.R:33]: `file_type` is also probably also a good place to use an R option to set the default

**Comment by stillmatic** [comment]
Cool - thanks for the PR Trevor.

I agree in principle with supporting feather instead of CSV - does it return different results, or is it just ugly? Might need to test a bit to see if we are losing data integrity using CSV as the intermediary. 

See comments inline re: `package_path`.

@akarve: the easiest path is if you include into the shell API a method for searching for a particular package, and returning the path to it. That would actually bypass a need to have a default search path in R, anyways - but also ensure consistency with Python.

**Review by stillmatic — COMMENTED** [review]
good overall, an opportunity to soft-code the package path and default intermediary type.

**Comment by tnederlof** [comment]
@stillmatic thank you for checking the code out. Was very easy to work with your existing code! Tonight I should have some time to look more at the difference between the feather output vs csv output (see how different the dataframes look). Will have to think of ways of testing the outputs on different data, etc as well. Will also think some more and wait to hear from @akarve regarding the package location (I saw your comment to him).

**Comment by stillmatic** [comment]
merging for now, #2 remains open.

**Pull request merged**